### DISCUSSION TITLE
YI 1080P Home vs YI 2k Pro Home

### DISCUSSION THREAD

**Discussion by TungstenE2** [roleoroleo/yi-hack-Allwinner-v2#845] [General]
Hi all,

last year I got 2 'YI 1080P Home' and they worked fine using the hack and pretty stabel on wifi.

Now I got 2 additional 'YI 2k Pro Home', did a firmware update via the app and installed the hack y623_0.3.1.

I see they are connected to wifi with good quality, but the Web Gui fails to open for one cam all the time and for the other in most cases.

They are in the same postion on my desk, where the YI 1080P Home worked very stable.

I also see that the vendor changed from 'AltoBeam (China) Inc.' to 'Shenzhen Bilian Electronic Co.，Ltd' for 'YI 2k Pro Home' and and 'Yi Dome Camera U (2K)'.
![grafik](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/39955685/ba7821e7-c5ad-4ca7-b342-49a85c6ea8b8)

What is your experience?

Is old 1080P version more stable than Pro versions?

thx

**Comment by roleoroleo** [comment]
> What is your experience?

I don't have this model, so I'm not able to give a feedback.
But it sounds strange.
Are the cam in bootloop? The ping is ok?

**Comment by TungstenE2** [comment]
I did now some testing for some weeks, but to me YFUSY with y623 does not run stable. Also updated to 0.3.2.
Stream is failing far too often, althought some thing was fixed by https://github.com/roleoroleo/yi-hack-Allwinner-v2/issues/846

Old cam version 1080p RFUSY using y231ga is much more stable.

May be related to new chipset and vendor.

I have 5 days remaining to send the 2k Pro version back to Amazon and I think I will do so.

**Reply by roleoroleo** [reply]
In 5 days I'm not able to fix these problems.
But if you decide to keep it, please tell me. I will work on this cam.

**Comment by TungstenE2** [comment]
thx, already send it back.

but please keep working on it. In germany the old version is no longer sold and this 2k version is advertized. So assume more people to own it. Would be great to have a stable version like the old 1080p version.
